.github/workflows/update-search-index.yml

```diff
@@ -0,0 +1,44 @@
+name: "Scheduled jobs: Update Seach Index"
+on:
+  schedule:
+    - cron:  '*/60 * * * *'
+  workflow_dispatch:
+jobs:
+  update:
+    name: Update Algolia Search Index
+    runs-on: ubuntu-latest
+    environment: testing
```

Comment:
we will only publish the index to the testing index in Algolia now, to confirm things there first where we can compare to prod.